employee/management/tests/test_delete_dummy_employees.py

```diff
@@ -0,0 +1,25 @@
+"""Unit tests for Delete dummy employees command."""
+
+from io import StringIO
+
+from django.core.management import call_command
+from django.test import TestCase
+
+from employee.models import Employee
+from tms.constants import DUMMY_EMAIL_MARKER
+from tms.utils import TestDbSetUp
+
+
+class DeleteDummyEmployeesTest(TestCase, TestDbSetUp):
+    """Dummy employees deletion test class."""
+
+    def setUp(self):
+        """Setup employee for deletion."""
+        self.employee = self.create_employee()
```

Comment:
why not sending marker here ?

Reply:
It's already using dummy marker in utils, no need to send from here.